src/utils/redeploy.ts

```diff
@@ -1,14 +1,17 @@
 import {
-  decode,
-  json,
   rest,
   setApplicationId,
   upsertSlashCommands,
 } from "../../deps.ts";
-import { commands } from "../commands/mod.ts";
-import translate from "../languages/translate.ts";
+import {  } from "sift/mod.ts";
```

Comment:
Why is this import empty? 👀

Reply:
I couldn't find { rest, setApplicationId, upsertSlashCommands } in any dependencies.. but I wrote it down because I thought a similar function would be in shift.